install.sh

```diff
@@ -417,7 +420,7 @@ os=$(uname_os)
 arch=$(uname_arch)
 format=$(get_format_name "${os}" "${arch}" "tar.gz")
 download_dir=$(mktemp -d)
-supported_tools="gensbom valint"
+supported_tools="valint gensbom"
```

Comment:
why the order was changed?

Reply:
For my own sanity, its faster to debug ...